rubicon/objc/types.py

```diff
@@ -619,106 +687,94 @@ def compound_value_for_sequence(seq, tp):
+    _UIEdgeInsetsEncoding = b"{UIEdgeInsets=dddd}"
+    _NSEdgeInsetsEncoding = b"{NSEdgeInsets=dddd}"
+    _PyObjectEncoding = b"^{_object=q^{_typeobject}}"
 else:
     c_ptrdiff_t = c_int
     NSInteger = c_int
     NSUInteger = c_uint
     CGFloat = c_float
-    _NSPointEncoding = b'{_NSPoint=ff}'
-    _CGPointEncoding = b'{CGPoint=ff}'
-    _NSSizeEncoding = b'{_NSSize=ff}'
-    _CGSizeEncoding = b'{CGSize=ff}'
-    _NSRectEncoding = b'{_NSRect={_NSPoint=ff}{_NSSize=ff}}'
-    _CGRectEncoding = b'{CGRect={CGPoint=ff}{CGSize=ff}}'
-    _NSRangeEncoding = b'{_NSRange=II}'
-    _UIEdgeInsetsEncoding = b'{UIEdgeInsets=ffff}'
-    _NSEdgeInsetsEncoding = b'{NSEdgeInsets=ffff}'
-    _PyObjectEncoding = b'^{_object=i^{_typeobject}}'
+    _NSPointEncoding = b"{_NSPoint=ff}"
+    _CGPointEncoding = b"{CGPoint=ff}"
+    _NSSizeEncoding = b"{_NSSize=ff}"
+    _CGSizeEncoding = b"{CGSize=ff}"
+    _NSRectEncoding = b"{_NSRect={_NSPoint=ff}{_NSSize=ff}}"
+    _CGRectEncoding = b"{CGRect={CGPoint=ff}{CGSize=ff}}"
+    _NSRangeEncoding = b"{_NSRange=II}"
+    _UIEdgeInsetsEncoding = b"{UIEdgeInsets=ffff}"
+    _NSEdgeInsetsEncoding = b"{NSEdgeInsets=ffff}"
+    _PyObjectEncoding = b"^{_object=i^{_typeobject}}"
 
 register_preferred_encoding(_PyObjectEncoding, py_object)
 
 
-@with_preferred_encoding(b'^?')
+@with_preferred_encoding(b"^?")
 # Anonymous structs/unions with unknown fields can't be decoded meaningfully,
 # so we treat pointers to them as unknown pointers.
-@with_encoding(b'^{?}')
-@with_encoding(b'^(?)')
+@with_encoding(b"^{?}")
+@with_encoding(b"^(?)")
 class UnknownPointer(c_void_p):
-    """Placeholder for the "unknown pointer" types ``^?``, ``^{?}`` and ``^(?)``.
+    """Placeholder for the "unknown pointer" types ``^?``, ``^{?}`` and
+    ``^(?)``.
 
     Not to be confused with a ``^v`` void pointer.
 
-    Usually a ``^?`` is a function pointer, but because the encoding doesn't contain the function signature,
-    you need to manually create a CFUNCTYPE with the proper types, and cast this pointer to it.
+    Usually a ``^?`` is a function pointer, but because the encoding doesn't
+    contain the function signature, you need to manually create a CFUNCTYPE with
+    the proper types, and cast this pointer to it.
 
-    ``^{?}`` and ``^(?)`` are pointers to a structure or union (respectively) with unknown name and fields. Such a type
-    also cannot be used meaningfully without casting it to the correct pointer type first.
+    ``^{?}`` and ``^(?)`` are pointers to a structure or union (respectively)
+    with unknown name and fields. Such a type also cannot be used meaningfully
+    without casting it to the correct pointer type first.
     """
 
 
 # from /System/Library/Frameworks/Foundation.framework/Headers/NSGeometry.h
 @with_preferred_encoding(_NSPointEncoding)
 class NSPoint(Structure):
-    _fields_ = [
-        ("x", CGFloat),
-        ("y", CGFloat)
-    ]
+    _fields_ = [("x", CGFloat), ("y", CGFloat)]
```

Comment:
This is more readable IMO. The trailing comma should stop Black from collapsing it onto a single line again.

```suggestion
    _fields_ = [
        ("x", CGFloat),
        ("y", CGFloat),
    ]
```